solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1523 @@
+    ///      delay elapses.
+    function finalizeDepositDustThresholdUpdate() external onlyOwner {
+        (, uint64 depositTreasuryFeeDivisor, uint64 depositTxMaxFee) = bridge
+            .depositParameters();
+        bridge.updateDepositParameters(
+            depositData.getNewDepositDustThreshold(),
+            depositTreasuryFeeDivisor,
+            depositTxMaxFee
+        );
+        depositData.finalizeDepositDustThresholdUpdate(governanceDelays[0]);
+    }
+
+    /// @notice Begins the deposit treasury fee divisor amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newDepositTreasuryFeeDivisor New deposit treasury fee divisor.
+    function beginDepositTreasuryFeeDivisorUpdate(
+        uint64 _newDepositTreasuryFeeDivisor
+    ) external onlyOwner {
+        depositData.beginDepositTreasuryFeeDivisorUpdate(
+            _newDepositTreasuryFeeDivisor
+        );
+    }
+
+    /// @notice Finalizes the deposit treasury fee divisor amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeDepositTreasuryFeeDivisorUpdate() external onlyOwner {
+        (uint64 depositDustThreshold, , uint64 depositTxMaxFee) = bridge
+            .depositParameters();
+        // slither-disable-next-line reentrancy-no-eth
+        bridge.updateDepositParameters(
+            depositDustThreshold,
+            depositData.getNewDepositTreasuryFeeDivisor(),
+            depositTxMaxFee
+        );
+        depositData.finalizeDepositTreasuryFeeDivisorUpdate(
+            governanceDelays[0]
+        );
+    }
+
+    /// @notice Begins the deposit tx max fee amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newDepositTxMaxFee New deposit tx max fee.
+    function beginDepositTxMaxFeeUpdate(uint64 _newDepositTxMaxFee)
+        external
+        onlyOwner
+    {
+        depositData.beginDepositTxMaxFeeUpdate(_newDepositTxMaxFee);
+    }
+
+    /// @notice Finalizes the deposit tx max fee amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeDepositTxMaxFeeUpdate() external onlyOwner {
+        (
+            uint64 depositDustThreshold,
+            uint64 depositTreasuryFeeDivisor,
+
+        ) = bridge.depositParameters();
+        // slither-disable-next-line reentrancy-no-eth
+        bridge.updateDepositParameters(
+            depositDustThreshold,
+            depositTreasuryFeeDivisor,
+            depositData.getNewDepositTxMaxFee()
+        );
+        depositData.finalizeDepositTxMaxFeeUpdate(governanceDelays[0]);
+    }
+
+    // --- Redemption
+
+    /// @notice Begins the redemption treasury fee divisor amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newRedemptionDustThreshold New redemption treasury fee divisor.
```

Comment:
Begins the redemption treasury fee divisor amount update process.
-> Begins the redemption dust threshold amount update process.

Reply:
[2056584](https://github.com/keep-network/tbtc-v2/pull/329/commits/205658478a76695bbe0f822cea8182268872cc05)